pom.xml

```diff
@@ -41,22 +41,33 @@
         <module>securebanking-openbanking-uk-rs-simulator-server</module>
         <module>securebanking-openbanking-uk-rs-simulator-sample</module>
     </modules>
+
     <parent>
         <groupId>com.forgerock.securebanking</groupId>
         <artifactId>securebanking-parent</artifactId>
         <version>1.0.4-SNAPSHOT</version>
     </parent>
 
+    <properties>
+        <uk.bom.version>1.1.3-SNAPSHOT</uk.bom.version>
+        <common.cors.version>1.0.0-SNAPSHOT</common.cors.version>
+    </properties>
```

Comment:
SNAPSHOT version? Release the cors lib first?

Reply:
We are working at the moment with SNAPSHOT, we don't have any release yet for any artifact.
One task that we need to do when we decide to release the first release will be define the strategy to release sbat.
At the moment all versions are SNAPSHOTS.

Reply:
Oh yeah! Doh